wpilibj/src/main/java/edu/wpi/first/wpilibj/PIDController.java

```diff
@@ -56,13 +57,20 @@
   private Tolerance m_tolerance;
   private double m_setpoint = 0.0;
   private double m_prevSetpoint = 0.0;
+  @SuppressWarnings("PMD.UnusedPrivateField")
```

Comment:
Why is this needed?

Reply:
PMD thinks (incorrectly) that `m_error` is an unused private variable because it is no longer read in `calculate()`.